### PR TITLE
MR-1146 d3 zoom event fix

### PR DESCRIPTION
**Pull request by vercasson**
- [x] This change is documented in CHANGELOG.md

### CODE REVIEW

**Comment by vercasson** [comment]
@wavebeem Any thoughts on how the changelog zoom related note should read?

**Comment by wavebeem** [comment]
> 
> 
> @wavebeem Any thoughts on how the changelog zoom related note should read?

I couldn't reproduce this in production. I don't think we ever released this bug. So it shouldn't be mentioned in the changelog.

File: src/MapComponent.tsx [hunk 1]
```diff
@@ -469,7 +469,7 @@ class MapComponent extends Component<MapComponentProps, MapComponentState> {
         // Don't destructure this at the top of the file because we need d3 to
         // hook until whatever the latest version of the function is, even if it
         // has changed since this callback was registered
-        .filter(() =>
+        .filter(d3Event =>
           this.props.shouldMapPanZoom
             ? this.props.shouldMapPanZoom(d3Event)
```
Comment (wavebeem) [inline src/MapComponent.tsx:474]: This changes the type of `shouldMapPanZoom`, but you ignored the difference by using `any` in the definition of `restrictedPanZoom`. I think you should update the type in `web-sdk.tsx` to match whatever the type of `d3Event` is here

**Review by wavebeem — CHANGES_REQUESTED** [review]

File: src/MapComponent.tsx (hunk 1 above)
Comment (vercasson) [reply src/MapComponent.tsx:474]: The type here for the event is `any` as specified by d3-zoom. They used `any` and made a comment on why.

**Review by wavebeem — APPROVED** [review]

**Pull request merged**